modules/rosa-sts-byo-oidc.adoc

```diff
@@ -1,29 +1,36 @@
 // Module included in the following assemblies:
 //
-// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+// * rosa_architecture/rosa-oidc-overview.adoc
 // * rosa_architecture/rosa-sts-about-iam-resources.adoc
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc
+// * rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.adoc
 
 ifeval::["{context}" == "rosa-hcp-sts-creating-a-cluster-quickly"]
 :rosa-hcp:
 endif::[]
-ifeval::["{context}" == "rosa-sts-about-iam-resources"]
-:rosa-classic:
-endif::[]
 
 :_content-type: PROCEDURE
-[id="rosa-hcp-byo-oidc_{context}"]
+[id="rosa-sts-byo-oidc_{context}"]
 = Creating an OpenID Connect configuration
 
-When using a {hcp-title} cluster, you must create the OpenID Connect (OIDC) configuration prior to creating your cluster. This configuration is registered to be used with OCM.
+When using a 
+ifdef::rosa-hcp[]
+{hcp-title} cluster, you must 
+endif::rosa-hcp[]
+ifndef::rosa-hcp[]
+{product-title} cluster, you may 
```

Comment:
Potential nit, here's what the ISG says about using the word "can": 
`Use to indicate ability. Use instead of could. To indicate possibility, use might. To indicate permission, use may.`

If "may" in this sentence is meant to indicate possibility instead of permission, I'd replace it with "can". Otherwise disregard this comment.